packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -398,25 +408,24 @@ public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
 
   private void onSignInResult(GoogleSignInResult result) {
     if (result.isSuccess()) {
-      finishWithSuccess(getSignInResponse(result.getSignInAccount()));
-    } else {
+      GoogleSignInAccount account = result.getSignInAccount();
+      Map<String, Object> response = new HashMap<>();
+      response.put("displayName", account.getDisplayName());
```

Comment:
Synced offline. Map and json implementations allow null in the response so this should be OK.